src/main/java/com/github/manolo8/darkbot/utils/LogUtils.java

```diff
@@ -20,20 +25,36 @@ public class LogUtils {
     public static final Path LOG_FOLDER = Paths.get("logs");
     public static final String START_TIME = LocalDateTime.now().format(FILENAME_DATE);
 
-    public static void setOutputToFile() {
+    public static void setOutputToFileAndConsole() {
         if (!Files.exists(LOG_FOLDER)) createFolder();
         else removeOld();
 
         try {
-            PrintStream output = getLogger();
-            System.setOut(output);
-            System.setErr(output);
-        } catch (FileNotFoundException | UnsupportedEncodingException e) {
-            System.out.println("Failed to redirect logs, file not found:");
+            OutputStream fileLogger = new FileOutputStream("logs/" + START_TIME + ".log");
+
+            System.setOut(createPrintStream(new FileOutputStream(FileDescriptor.out), fileLogger));
+            System.setErr(createPrintStream(new FileOutputStream(FileDescriptor.err), fileLogger));
+        } catch (FileNotFoundException e) {
+            System.out.println("Failed to redirect logs, file not found: " + "logs/" + START_TIME + ".log");
             e.printStackTrace();
         }
     }
 
+    private static PrintStream createPrintStream(FileOutputStream multi, OutputStream fileLogger) {
+        OutputStream multiOut = new MultiOutputStream(multi, fileLogger);
+        OutputStream bufferedOut = new BufferedOutputStream(multiOut, 128);
+
+        try {
+            if (System.getProperty("file.encoding").equals(Charset.defaultCharset().toString()))
+                return new PrintStreamWithDate(bufferedOut, "UTF-8");
```

Comment:
Don't hard-code utf-8, respect the OS's encoding.
In windows the default isnt utf8 becsuse the windows console doesnt support it, and it would break with this change.